Cargo.toml

```diff
@@ -10,14 +10,21 @@ repository = "https://github.com/prometheus/client_rust"
 homepage = "https://github.com/prometheus/client_rust"
 documentation = "https://docs.rs/prometheus-client"
 
+[features]
+protobuf = ["dep:prost", "dep:prost-types", "dep:prost-build", "dep:prometheus-client-derive-proto-encode", "dep:void"]
```

Comment:
I think CI is currently not running with `--all-features`. Can you add it with this pull request to make sure your feature is being tested?

Reply:
https://github.com/prometheus/client_rust/pull/47/commits/33c940a6e7f8d95fbf4479c6e2301f40bff4670c